LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2017-2019 Uber Technologies, Inc.
```

Comment:
I think you want to change it to
Copyright (c) 2017 Uber Technologies, Inc.

as the second number might mean the last date the copyright applies.